contrib/intel/jenkins/tests.py

```diff
@@ -688,58 +693,100 @@ def execute_cmd(self):
 class MpichTestSuite(Test):
 
     def __init__(self, jobname, buildno, testname, core_prov, fabric,
-                 hosts, mpitype, ofi_build_mode, user_env, log_file, util_prov=None):
+                 hosts, mpitype, ofi_build_mode, user_env, log_file, util_prov=None, weekly=None):
 
         super().__init__(jobname, buildno, testname, core_prov,
                          fabric, hosts, ofi_build_mode, user_env, log_file, mpitype,
                          util_prov)
-
-        self.mpichsuitepath = f'{self.middlewares_path}/{mpitype}/'\
-                              'mpichsuite/test/mpi/'
+        self.mpichpath = f'{cloudbees_config.build_dir}/mpich_temp/mpich-4.1/'
```

Comment:
can this be mpich_temp/mpich and extract the tar in build.py to rename it to be mpich? That way when we upgrade in the future it requires less changes